feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -0,0 +1,395 @@
+# TE-18.1 gRIBI MPLS in UDP Encapsulation with OC QoS scheduler
+
+Create AFT entries using gRIBI to match destination IP address in a
+network-instance.  Encapsulate the matching packets in MPLS in UDP. Configure
+a qos scheduler using gNMI to rate limit / police traffic based on matching
+the destination IP address of a packet input to the DUT.
+
```

Comment:
Add encapsulation for ethernet packets with mpls-in-udp IPv6 (in a separate README).

Encap all packets received in a VRF and send to statically configured IP (up to 16) (NHG) - ie: this is policy-forwarding in OC.

Reply:
> Add encapsulation for ethernet packets with mpls-in-udp IPv6 (in a separate README).
> 
> Encap all packets received in a VRF and send to statically configured IP (up to 16) (NHG) - ie: this is policy-forwarding in OC.

I think above comment refers the same.

Reply:
done